CONTRIBUTING.md

```diff
@@ -16,11 +16,11 @@ read/subscribe to the following resources:
 ## What to work on
 
 You should work on what you want to. Our bug tracker is
-here: https://joindin.jira.com
+here: https://github.com/joindin/joindin-web2/issues
 
 Any issues that have the "hackathon" or "easypick" label are ones that we think
-are a good starting point. This [JIRA filter](https://joindin.jira.com/issues/?jql=project%20%3D%20JOINDIN%20AND%20labels%20in%20(hackathon%2C%20%22OR%22%2C%20easypick)) will give you the list of all
-current issues with the "hackathon" or "easypick" issues.
+are a good starting point. This [filter](https://github.com/joindin/joindin-web2/issues?q=is%3Aopen+is%3Aissue+label%3Aeasypick) will give you the list of all
```

Comment:
While the jira-link referenced *all* the easypicks for the joind.in-project this now references only the easy-picks for web2. Do we want to have easypicks for the whole project?

Reply:
https://github.com/orgs/joindin/projects/1

Reply:
Says didn't find anything here for me ?